config/filter.d/lighttpd-auth.conf

```diff
@@ -4,7 +4,13 @@
 [Definition]
 
 failregex = ^: \(http_auth\.c\.\d+\) (password doesn\'t match .* username: .*|digest: auth failed for .*: wrong password|get_password failed), IP: <HOST>\s*$
+            ^: \(\S+\.c\.\d+\) user \S* ?(not found|not authorized|password mismatch|sent invalid token|misbehaved): .+, IP: <HOST>
```

Comment:
If there is anything from a client in that message (eg login) and there is no trailing anchor - opens possibility for the injection command